### DISCUSSION TITLE
유니티로 제작시 제페토 기본 UI의 적용이 가능한지 여쭤보고 싶습니다.

### DISCUSSION THREAD

**Discussion by zepzino** [naverz/zepeto-studio-kor#422] [5. Q&A]
가능한 기능이 어떤 것이고 불가능한 기능이 어떤 것인지 아직 모르겠네요 ㅠㅠ
유니티로 제작할경우 UI가 달라지면 사용이 불편할 것 같은데.. 과연 유니티로 해도 괜찮을지 걱정입니다.

고수분들의 답변을 부탁드립니다 ㅠㅠㅠ

![KakaoTalk_Photo_2022-02-23-18-07-27](https://user-images.githubusercontent.com/100274420/155290822-96c7263c-d4ab-4fed-93dc-88d62a0a12ca.jpeg)

**Comment by x4cf** [comment]
File.IO 와 외부네트워크접속 기능은 사용 불가능해요 :D
사용불가능한 기능을 제외한 모든 기능은, 지원하는 함수를 이용해 직접 제작해야합니다.
